### ISSUE TITLE
Java: Android Studio: ROSjava publishing to a topic based on button input

### ISSUE THREAD

**Issue by PercivalChan** [rosjava/rosjava_core#309]
This is my first time come into ROS android-related things. I am currently working on the android app side of a ROS project. I have altered the ROSandroid project from git. I added a Bluetooth scanning function fragment in the app. I am required to detect Bluetooth beacon (Which is done) and send message like "1", "2" to let the robot recognize the beacon number. In my part, I just have to communicate with the ROS after the Bluetooth scan by pressing the button "Send Car" in the BT fragment.
![WhatsApp Image 2022-08-05 at 12 00 20 PM](https://user-images.githubusercontent.com/110666676/183053696-8f081815-2eaa-48b6-8272-d0bb05e87a37.jpeg)

However, I failed to extract the code of node functions from the source code of ROSandroid. Does anyone know how to retrieve the attributes and functions so I can make use of the Master IP and send messages through node? (Or, alternatively, how can I send message to ROS OS in android 12 environment? if available) Example: https://answers.ros.org/question/272979/rosjava-publishing-to-a-topic-based-on-user-input-button/

Also, when I reach out some past tutorials, I failed to run all of them as nearly all of the examples or tutorials regarding this topic are outdated. (At least 4 years ago)

To sum up, I would like to get IP address from ROSandroid app, then scan Bluetooth beacon, lastly send a message to inform the ROS the beacon number. So I just need to know how can android 12 publish message through node.

There is still a lot to learn. Please give me guidance if it is convenient. Thank you much for your time.
